app/views/content/landing/how-much-do-teachers-get-paid/_content.html.erb

```diff
@@ -0,0 +1,55 @@
+<div class="row">
+  <section class="col col-720">
+    <h2 class="heading--box-blue">More about teachers' salaries and benefits</h2>
+
+    <div class="inset">
+      <p>
+        Your teaching salary will depend on your role in the school and qualifications.
+        There are lots of opportunities to increase your salary during your career:
+      </p>
+
+      <ul>
+        <li>teachers who make a sustained contribution to their school could expect to earn over £40k</li>
+        <li>headteachers are usually the highest paid people in a school and can expect to earn around £50k to £131k depending on the school and location</li>
+      <p><%= link_to("Find out more about teaching salaries and benefits", page_path("salaries-and-benefits")) %>.</p>
```

Comment:
@philspray @EllieNodder  The link looks like it is within the bullet list - I think this is because there is no </ul>?